frappe/patches.txt

```diff
@@ -201,6 +200,7 @@ execute:frappe.reload_doc("desk", "doctype", "Form Tour")
 [post_model_sync]
 execute:frappe.get_doc('Role', 'Guest').save() # remove desk access
 frappe.core.doctype.role.patches.v13_set_default_desk_properties
+frappe.patches.v14_0.update_workspace2 # 06.06.2023
```

Comment:
Is re-executing the patch required on all existing sites?

@shariquerik ?

Reply:
Nope